src/app/redux/reducers/commonReducers.js

```diff
@@ -72,6 +75,25 @@ const commonReducer = (state = initalState, action) => {
           allAdminUsersFailure: true,
         },
       };
+    case types.GET_ALL_ADMIN_USERS_IN_DEPT_SUCCESS:
```

Comment:
i see only success and failure scenarios. Please add a loading condition as well

Reply:
Added loading condition